src/promptflow/promptflow/_cli/_pf/_tool.py

```diff
@@ -95,3 +127,10 @@ def init_tool(args):
     ToolPackageGenerator(tool_name=args.tool).generate_to_file(script_code_path / f"{args.tool}.py")
     InitGenerator().generate_to_file(script_code_path / "__init__.py")
     print(f'Done. Created the tool "{args.tool}" in {script_code_path.resolve()}.')
+
+
+@exception_handler("Tool list")
+def list_tool(args):
+    pf_client = PFClient()
+    package_tools = pf_client._tools.list(args.flow)
```

Comment:
Is tools operations public, like use it as  `pf.tools.list`?

Reply:
tool operation is currently private and will not be exposed to user